lib/idl/idl_checkers.es6.js

```diff
@@ -65,5 +65,27 @@ module.exports = [
     logger.info('Checking top-level member names');
 
     eachKey(logger, left.members || [], right.members || [], 'name');
-  }
+  },
+  function rightMissingConstructor(logger, left, right) {
+    function isCtor(attr) {
+      return attr.name === 'Constructor';
+    }
+    function ctorCount(idl) {
+      return idl.attrs ? idl.attrs.filter(isCtor).length : 0;
+    }
+    const chk = check.bind(this, logger);
+    if (ctorCount(left) > ctorCount(right)) {
+      const lcs = JSON.stringify(left.attrs.filter(isCtor), null, 2);
+      const rcs = JSON.stringify(
+        right && right.attrs ? right.attrs.filter(isCtor) : [], null, 2
+      );
```

Comment:
This style has pervaded this repo. We can do a fix-up CL later if we want to better align it with Chromium style.